core/chains/evm/txmgr/eth_confirmer.go

```diff
@@ -730,7 +730,7 @@ func (ec *EthConfirmer) attemptForRebroadcast(ctx context.Context, lggr logger.L
 		}
 		attempt, err = ec.bumpGas(ctx, etx, etx.EthTxAttempts)
 
-		if gas.IsBumpErr(err) {
+		if gas.IsBumpErr(err) || errors.Is(err, gas.ErrConnectivity) {
```

Comment:
Can you move this ErrConnectivity error to inside the IsBumpErr()?
The connectivity error means that we are not allowing bumping, so looks to me the same class.
Makes the logic inside confirmer doesn't need to know about this chain specific error.